EIPS/eip-4626.md

```diff
@@ -87,17 +87,13 @@ MUST return the address of a token implementing the ERC-20 standard.
 #### calculateShares
 `function calculateShares(uint256 underlyingAmount) public view returns (uint256 shareAmount)`
 
-Returns the amount of vault shares corresponding to a given underlying amount.
-
-`calculateShares(calculateUnderlying(shareAmount))` MUST equal `shareAmount`
+Returns the amount of vault shares corresponding to a given underlying amount, for the `withdraw` method. The return value is only guaranteed to be exact if executed immediately before a `withdraw` call in the same transaction.
```

Comment:
not following, why `withdraw` specifically? and wouldnt it be immediately **after**?